src/libraries/System.Net.Http.WinHttpHandler/ref/System.Net.Http.WinHttpHandler.cs

```diff
@@ -48,4 +48,10 @@ public partial class WinHttpHandler : System.Net.Http.HttpMessageHandler
         protected override void Dispose(bool disposing) { }
         protected override System.Threading.Tasks.Task<System.Net.Http.HttpResponseMessage> SendAsync(System.Net.Http.HttpRequestMessage request, System.Threading.CancellationToken cancellationToken) { throw null; }
     }
+
+    public enum HttpKeepAlivePingPolicy
```

Comment:
This is weird. This enum is there as it is used in HttpHandlerDefaults. 
Only way how to prevent it what I have in my mind is make HttpHandlerDefaults partial and move SocketsHttpHandler related properties to System.Net.Http only.

Reply:
Definitely split it, this shouldn't be in `WinHttpHandler` at all.

Reply:
done